api/Dockerfile

```diff
@@ -2,8 +2,10 @@ FROM python:3.7
 
 WORKDIR /app
 RUN pip install watchgod
```

Comment:
The `FROM python:3.7` to `FROM python:3.8`

Reply:
Yep.